x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/fields/code_editor.tsx

```diff
@@ -57,7 +57,7 @@ export const CodeEditor = ({
           aria-label={ariaLabel}
           data-test-subj="codeEditorContainer"
         >
-          <MonacoCodeEditor
+          <MonacoCodeEditor // TODO theme provided in context by x-pack/plugins/synthetics/public/apps/synthetics/synthetics_app.tsx
```

Comment:
Should we create a follow up issue for this?

Reply:
Is it failing to render? Based on my search (with limited understanding of that plugin 😅 ), it looked like `theme` was being provided.

If it is throwing an error, we can fix it here, or, if it is a bigger change we can create a new issue

Reply:
Heya @shahzad31! Can you verify these `CodeEditor`s aren't rendering? Or would you rather create another issue to address this later?

Reply:
it seems to be working 
<img width="1390" alt="image" src="https://github.com/elastic/kibana/assets/3505601/767e45b9-cad5-46ec-b65a-d952dc8850d3">